js/ui/scheduler/ui.scheduler.appointments.strategy.horizontal_month_line.js

```diff
@@ -11,14 +11,17 @@ var HorizontalMonthLineRenderingStrategy = HorizontalAppointmentsStrategy.inheri
 
         startDate = dateUtils.trimTime(startDate);
 
-        var durationInHours = (endDate.getTime() - startDate.getTime()) / 3600000,
-            width = Math.ceil(durationInHours / 24) * cellWidth;
-
+        var width = Math.ceil(this.getDurationInHour(startDate, endDate) / 24) * cellWidth;
```

Comment:
1) rename to `_getDurationInHours` to mark that your function is private
2) use const for `24`